docs/user/plugins/cloudFoundry/cloud-foundry-plugin.md

```diff
@@ -0,0 +1,99 @@
+# Cloud Foundry Plugin
+
+## Introduction
+The Cloud Foundry Plugin transforms a CSAR to an artifact which can be deployed on the Cloud Foundry Platform. In general the following steps have to be done to transform and deploy a CSAR:
+
+1. upload CSAR with the GUI/ CLI
+2. select Platform CloudFoundry
+3. enter required Inputs
+4. start Transformation
+5. download artifact
+6. unzip at target location
+7. start deploying by executing the deploy.sh found in output/scripts/ in target artifact
+
+Before the Transformation can start the Cloud Foundry credentials and additional properties have to be set:
+
+- `CF_Instance_Username`: Username of CloudFoundry user account
+- `CF_Instance_Api`: The endpoint of the provider
+- `CF_Instance_Organization`: The organization of the user account which should be used for deployment
+- `CF_Instance_Password`: Password of CloudFoundry user account
+- `CF_Instance_Space`: The space of the user account which should be used for deployment
+
+The credentials are used to get a list of services which can be used for deployment.
+For more details on supported NodeTypes, the basics of Cloud Foundry, the Transformation and the created scripts, see the documents in `dev/plugins/cloudFoundry`. The deployment of an artifact was mainly tested with Pivotal Cloud Foundry.
+
+## Artifact
+The Artifact is the result of the transformation which contains all necessary files which are needed to deploy an application on Cloud Foundry. After you open the Artifact you can see the log of the Transformation, the app folder(s) which contain your application and the output folder which contains the files for deployment. If you provide multiple applications in your CSAR the artifact will contain multiple app folders called app1, app2 and so on. To get a better understanding what applications means in the context of Cloud Foundry plugin, see section `CloudFoundry application` in `dev/plugins/cloudFoundry/How_add_new_Node_Types`.
+
+![Artifact overview](img/artifact-overview.png)
+
+App folder: The .bp-config folder in this case contains buildpack additions for a php application, which needs some additional buildpacks.
+
+![Artifact app folder](img/artifact-app.png)
+
+The output folder contains a scripts folder and the files all_services.txt, manifest.yml and README.txt.
+
+![Artifact output folder](img/artifact-output.png)
+
+Scripts folder:
+
+![Artifact scripts folder](img/artifact-scripts.png)
+
+The Plugin tries to find free services available on the Cloud Foundry Platform to provide support for your modeled NodeTemplates. They can be changed to paid plans. To see what additions a paid plan brings, please check your Cloud Foundry provider. You can see a list of all available services and corresponding plans in all_services.txt in the output folder.
```

Comment:
Consider using:
"To see what additions a paid plan brings..." -> "What benefits will bring changing to a paid plan..."